_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -2,54 +2,142 @@
 layout: page
 title: Add a Log Drain to your App
 nav: Log Drain
-modified_at: 2019-11-18 00:00:00
+modified_at: 2020-05-22 00:00:00
 tags: Papertrail Logentries Logmatic Graylog logs integration
 ---
 
-{% include upon_request.md %}
-
 ## Integration Purposes
 
-When an application starts having a higher traffic, reading logs can be cumbersome. Using a log
-management solution to gather the logs from all your applications and all your containers eases your
-work when trying to find an information amongst all these lines of logs. A few SaaS solutions exist
-such as Logentries, Logmatic.io, Papertrail or Graylog. You can also host such solution on Scalingo.
-Here is a tutorial on [how to deploy the ELK
-stack](https://scalingo.com/articles/2018/02/23/running-the-elk-stack-on-scalingo.html) on Scalingo.
-We also support sending your applications logs using the standard syslog protocol over UDP, TCP or
-TCP+TLS.
+Using a log management solution to gather the logs from applications ease the
+work when trying to find an information.
+A few SaaS solutions exist such as Logentries, Logmatic.io, Papertrail or
+Graylog. You can also host such solution on Scalingo. Here is a tutorial on
+[how to deploy the ELK
+stack](https://scalingo.com/articles/2018/02/23/running-the-elk-stack-on-scalingo.html)
+on the platform.
+
+Scalingo also supports sending applications logs using the standard
+syslog protocol over UDP, TCP or TCP+TLS.
+
+{% note %}
+Adding a log drain to an application will not disable
+[scalingo logs](https://doc.scalingo.com/platform/app/logs)
+{% endnote %}
 
-## Third-party Provider integrations
+## Third-Party Provider Integrations
 
-Our system is also able to send logs to the following SaaS log processing providers:
+Our system is able to send logs to the following SaaS logs processing providers:
 
-* Scalingo hosted ELK stack ([how to setup]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %}))
-* Datadog
-* Logentries
-* Graylog
-* Papertrail
-* Syslog
+* [Scalingo hosted ELK stack](#scalingo-hosted-elk-stack)
+([how to setup]({% post_url platform/getting-started/2000-01-01-getting-started-with-elk %}))
+* [Datadog](#datadog)
+* [Logentries](#logentries)
+* [Graylog](#ovh-hosted-graylog)
+* [Papertrail](#papertrail)
+* [Syslog](#syslog-tcptls)
```

Comment:
I think we could add a word here about "What if my drain of choice is not supported ?". After this list of providers, adding a word stating that they can contact Scalingo support if they need another provider to be added.